features/editable/cmake/.gitignore

```diff
@@ -0,0 +1,3 @@
+build_debug
```

Comment:
This file works here in a subdirectory? 😮

Reply:
actually, I think I've copied by mistake